core/domain/skill_domain.py

```diff
@@ -378,3 +378,36 @@ def to_dict(self):
             'skill_model_created_on': self.skill_model_created_on,
             'skill_model_last_updated': self.skill_model_last_updated
         }
+
+
+class SkillsMastery(object):
+    """Domain object for an Oppia Skills Mastery."""
+
+    def __init__(self, user_id, skill_id, degree_of_mastery):
+        """Constructs a SkillMastery domain object.
+
+        Args:
+            user_id: str. The user id of the logged in user.
+            skill_id: str. The id of the skill.
+            degree_of_mastery: float. The progress of user for the
+                corresponding skill.
+        """
+        model_id = user_id + '-' + skill_id
```

Comment:
Since, you are finding this model_id in a lot of places, I'd recommend having a get_model_id() function (which returns this string) in this class and call it as self.get_model_id(user_id, skill_id).

Reply:
okay

Reply:
In skill_domain.py the variable is used just once. So I've added the function in only skills_services.py